src/main/scala/com/atomist/rug/runtime/plans/LocalPlanRunner.scala

```diff
@@ -47,11 +47,11 @@ class LocalPlanRunner(messageDeliverer: MessageDeliverer,
   private def handleInstruction(plannable: Plannable, callbackInput: Option[Response]): Seq[PlanLogEvent] = {
     Try { instructionRunner.run(plannable.instruction, callbackInput) } match {
       case ScalaFailure(error) =>
-        val msg = s"Failed to run instruction: ${plannable.instruction} - ${error.getMessage}"
-        logger.error(msg)
+        val msg = s"Failed to run ${plannable.toDisplay} - ${error.getMessage}"
+        logger.error(msg, error)
         Seq(InstructionError(plannable.instruction, error))
       case ScalaSuccess(response) =>
-        val msg = s"Ran instruction: ${plannable.instruction} and got response: $response"
+        val msg = s"Ran ${plannable.toDisplay} and then ${response.toDisplay}"
```

Comment:
Yeah, because "running" a message sounded dumb. Callbacks can be Messages, Plans, Or Respond instructions.